app/controllers/catalog_controller.rb

```diff
@@ -28,7 +28,7 @@ class CatalogController < ApplicationController
       qt: 'search',
       rows: 10,
       qf: 'title_tesim description_tesim creator_tesim keyword_tesim',
-      fq: '{!term f=has_model_ssim v=Work}'
+      # fq: '{!term f=has_model_ssim v=Work}'
```

Comment:
I'm afraid that if we remove this entirely we'll start getting content we don't want in search results, e.g., AdminSets, PermissionTemplates etc. Could we instead say the `has_model_ssim` should match either Work or Collection?

Or, if I'm wrong, and doing this does not lead to lots of extraneous content in the search results, let me know that. Maybe deploy this branch to ursus-dev and do an experiment?